javascript/detail_and_listview.js

```diff
@@ -226,15 +245,9 @@ var makeDetailView = function(id) {
         userImage.src = snapshot.val()['thumbnail'];
         var startLocation = snapshot.val()['startLocation'];
         var pois = snapshot.val()['pois'];
-        if (walkerLocation !== null){
```

Comment:
just double checking to see if these changes need to stay or that you removed them on purpose

Reply:
good point... I feel like I might be overriding someone else's changes... Let me double check

Reply:
good catch, we definately need to account for behavior if users don't volunteer their location.